.gitignore

```diff
@@ -128,3 +128,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# VSCode
+.vscode
```

Comment:
I'm wondering if we should use a global .gitignore to ignore IDE-related artifacts. What do you think?